source/channels.yaml

```diff
@@ -343,6 +343,39 @@
         '500':
           description: Could not retrieve the channel member, the user is not a member of the channel or the channel does not belong to the team
 
+  '/teams/{team_id}/channels/{channel_id}/members/ids':
+      post:
+        tags:
+          - channels
+        summary: Get channel members by IDs
+        description: Get a map of channel member objects for the specified channel and user IDs.
+        parameters:
+          - name: team_id
+            in: path
+            description: Team ID of the member to return
```

Comment:
Maybe just change this to `Team ID for the corresponding channel` or something similar?